### ISSUE TITLE
Event listener never removed

### ISSUE THREAD

**Issue by LinusU** [Sitebase/cssinjs#1]
You are adding a bound function but then removing the original function, it won't work.

``` javascript
function hello() { console.log('Hello'); }

el.addEventListener('mousein', hello.bind(this));
el.removeEventListener('mousein', hello); // Nothing will be removed
```

https://github.com/Sitebase/cssinjs/blob/feature-interaction-mixin/interaction-aware-mixin.js

**Comment by Sitebase** [comment]
Thanks for the heads up @LinusU 
How can this be solved?
Do I needs to store the bound version of the function or something?

**Comment by LinusU** [comment]
Yes, the same value that's being passed to `addEventListener` must be passed to `removeEventListener`.

If there is a function that will be called when the instance is created (haven't used mixins so I don't know) I would store the bound functions on the new instance. Then just use `this.onOver` and `this.onOut` directly.

``` javascript
constructor: function () {
  this.onOver = this.onOver.bind(this);
  this.onOut = this.onOut.bind(this);
}
```
